lntest/harness.go

```diff
@@ -1615,7 +1615,9 @@ func (n *NetworkHarness) sendCoins(amt btcutil.Amount, target *HarnessNode,
 		return err
 	}
 
-	expectedBalance := btcutil.Amount(initialBalance.ConfirmedBalance) + amt
+	fullInitialBalance := initialBalance.ConfirmedBalance +
+		initialBalance.UnconfirmedBalance
+	expectedBalance := btcutil.Amount(fullInitialBalance) + amt
```

Comment:
Thinking we might also eventually need this check elsewhere where we see certain balance related flakes.